Dockerfile

```diff
@@ -54,7 +54,7 @@ COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/dockerfiles
 COPY --from=builder /go/src/github.com/pressly/goose/cmd/goose/goose goose
 COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/db/migrations/* db/migrations/
 COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/plugins/transformerExporter.so plugins/transformerExporter.so
-COPY --from=builder /go/src/github.com/makerdao/vulcanizedb/dockerfiles/wait-for-it.sh .
+COPY --from=builder /go/src/github.com/makerdao/vdb-mcd-transformers/dockerfiles/wait-for-it.sh .
```

Comment:
Just curious, does this just determine what repo this file lives in?

Reply:
It determines what repo we pull the file from. We could technically continue to grab it from vulcanizedb, but that raises two issues:
- if the file doesn't exist in vdb (either an old version or because we decide to remove it), the build fails
- if we want to change that file, we need to do it in that repo